test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -1398,9 +1399,10 @@ protected void assertRowsNet(ProtocolVersion protocolVersion, ResultSet result,
             Object[] expected = rows[i];
             Row actual = iter.next();
 
-            Assert.assertEquals(String.format("Invalid number of (expected) values provided for row %d (using protocol version %s)",
-                                              i, protocolVersion),
-                                meta.size(), expected.length);
+            Assertions.assertThat(meta.size())
```

Comment:
functionally the same but was confused when I saw the error msg; it did not contain the row, and it was backwards so (expected = actual and actual = expected)